assets/sass/_app.scss

```diff
@@ -40,9 +39,13 @@ Tags:
 3.0 Variables
 --------------------------------------------------------------*/
 @import "components/variables";
+@import "components/content";
```

Comment:
@nucro the content, header, footer, and comments scss files should go in a folder named "modules" instead of "components".  And they should be listed in a new section 7.0.  please review this file for an example:
https://github.com/moxienyc/amtrak-journey/blob/develop/wp-content/themes/amtrak-journey/assets/sass/_app.scss